test/postman/identity-test-collection.json

```diff
@@ -0,0 +1,649 @@
+									"    pm.expect(jsonData.token_type).to.eql(\"bearer\");",
+									"});",
+									"",
+									"pm.test(\"Response contains access token\", function () {",
+									"    pm.expect(\"access_token\" in jsonData).to.be.true;",
+									"});",
+									"",
+									"var tokens = JSON.parse(responseBody); ",
+									"pm.environment.set(\"kcAccessToken\", tokens.access_token);",
+									""
+								]
+							}
+						}
+					],
+					"request": {
+						"method": "POST",
+						"header": [
+							{
+								"key": "Content-Type",
+								"value": "application/x-www-form-urlencoded"
+							}
+						],
+						"body": {
+							"mode": "urlencoded",
+							"urlencoded": [
+								{
+									"key": "client_id",
+									"value": "{{identityServiceClientId}}",
+									"type": "text"
+								},
+								{
+									"key": "grant_type",
+									"value": "password",
+									"type": "text"
+								},
+								{
+									"key": "username",
+									"value": "{{testUserUsername}}",
+									"type": "text"
+								},
+								{
+									"key": "password",
+									"value": "{{testUserPassword}}",
+									"type": "text"
+								}
+							]
+						},
+						"url": {
+							"raw": "{{identityServiceUrl}}/realms/{{identityServiceRealm}}/protocol/openid-connect/token",
+							"host": [
+								"{{identityServiceUrl}}"
+							],
+							"path": [
+								"realms",
+								"{{identityServiceRealm}}",
+								"protocol",
+								"openid-connect",
+								"token"
+							]
+						}
+					},
+					"response": []
+				}
+			]
+		},
+		{
+			"name": "02 ACS-token-auth",
+			"description": null,
+			"item": [
+				{
+					"name": "Retrieve test user profile using token",
+					"event": [
+						{
+							"listen": "test",
+							"script": {
+								"id": "cf5678ae-cc22-4e8c-967a-25809a0a6a1c",
+								"type": "text/javascript",
+								"exec": [
+									"pm.test(\"Response code is 200\", function () {",
+									"    pm.response.to.have.status(200);",
+									"});",
+									"",
+									"pm.test(\"Username in response is valid\", function () {",
+									"    var jsonData = pm.response.json();",
+									"    var userName= jsonData.entry.firstName;",
+									"    pm.expect(userName).to.be.eql(pm.variables.get(\"testUserUsername\"));",
+									"});",
+									"",
+									"pm.test(\"Id in response is valid\", function () {",
+									"    var jsonData = pm.response.json();",
+									"    var id = jsonData.entry.id;",
+									"    pm.expect(id).to.be.eql(pm.variables.get(\"testUserUsername\"));",
+									"});"
+								]
+							}
+						}
+					],
+					"request": {
+						"method": "GET",
+						"header": [
+							{
+								"key": "Authorization",
+								"value": "Bearer {{kcAccessToken}}"
+							}
+						],
+						"body": {
+							"mode": "raw",
+							"raw": ""
+						},
+						"url": {
+							"raw": "{{acsUrl}}/api/-default-/public/alfresco/versions/1/people/-me-",
```

Comment:
Won't all these ACS API tests fail when testing the identity service in isolation?

Reply:
That is one of the things Kris and I talked about.  Looking at the tests I think that there is really only one possible tests that would actually be copied. He was in agreement that we should copy all of them. I am good to cut them down to just the one.